contents/swift/welcome-to-swift.md

```diff
@@ -54,20 +70,18 @@ func yearAlbumReleased(name: String) -> Int? {
 }
 ```
 
-### Optional Binding
+## Optional Binding
```

Comment:
Shouldn't this be just a continuation of the previous section?

Reply:
Hmm as there's a change in topic (the previous section was on returning an optional value, while this section is on unwrapping an optional value), I thought that we should have some sort of segmentation to make it clearer.

The section below (`Guard Statements`) can also be seen of as part of `Optional` because guard statements are only used in the context of unwrapping optionals. 

As such, I'm thinking if we should actually use `###` for both `Optional Binding` and `Guard Statements`.